cs-commons/src/main/java/io/cloudslang/content/utils/NumberUtilities.java

```diff
@@ -0,0 +1,132 @@
+package io.cloudslang.content.utils;
+
+import io.cloudslang.content.constants.ExceptionsValues;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.math.NumberUtils;
+import org.jetbrains.annotations.Nullable;
+
+/** A Number utility class that offers integer and double validation and conversion.
+ * Created by victor on 31.08.2016.
+ */
+public final class NumberUtilities {
+
+    private NumberUtilities() {}
+
+    /** Given an integer string, it checks if it's a valid integer (based on apaches NumberUtils.createInteger)
+     * @param integerStr the integer string to check
+     * @return true if it's valid, otherwise false
+     */
+    public static boolean isValidInt(@Nullable final String integerStr) {
+        if (StringUtils.isBlank(integerStr)) {
+            return false;
+        }
+        final String stripedInteger = StringUtils.strip(integerStr);
+        try {
+            NumberUtils.createInteger(stripedInteger);
+            return true;
+        } catch (NumberFormatException e) {
+            return false;
+        }
+    }
+
+    /** Given an integer string, it checks if it's a valid integer (base on apaches NumberUtils.createInteger) and if
+     * it's between the lowerBound and upperBound (without including them)
+     * @param integerStr the integer string to check
+     * @param lowerBound the lower bound of the interval
+     * @param upperBound the upper bound of the interval
+     * @return true if the integer string is valid and in between the lowerBound and upperBound, false otherwise
+     * @throws IllegalArgumentException if the lowerBound is not less than the upperBound
+     */
+    public static boolean isValidInt(@Nullable final String integerStr, int lowerBound, int upperBound) {
```

Comment:
usually intervals are inclusive exclusive in Java, you can be even more generic by passing `includeLeftMargin `and `includeRightMargin`